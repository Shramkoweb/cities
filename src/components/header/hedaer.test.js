import React from "react";
import renderer from "react-test-renderer";
import Header from "./header";

it(`Header component render correct`, () => {
  const header = renderer
    .create(<Header user={`shramko.serhii@gmail.com`}/>)
    .toJSON();
  expect(header).toMatchSnapshot();
});
