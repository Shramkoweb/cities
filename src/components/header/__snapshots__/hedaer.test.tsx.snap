// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render is correctly if user already authorization 1`] = `ShallowWrapper {}`;

exports[`Header component should render is correctly if user require authorization 1`] = `ShallowWrapper {}`;
